recipes/cfgfile/0.2.8.2.1/test_package/example.cpp

```diff
@@ -0,0 +1,36 @@
+
+#include <string>
+#include <fstream>
+
+#include "cfg.hpp"
+#include "config.hpp"
+
+int main()
+{
+    Cfg::Session cfg;
+
+    std::ifstream stream( c_input_folder + "/test.cfg" );
```

Comment:
Since the layout of a test_package is standard could we cut the complexity down?

Could we either
1. pass the path as a command line argument
2. just hardcode the `../test.cfg` since we always run in from `bin/`?